ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1367,6 +1370,96 @@ where is_in_physics_bubble(vessel)
                                                 body.atmosphereDepth));
   }
 
+  private void RemoveBuggyTidalLocking() {
+    ApplyToBodyTree(body => body.tidallyLocked = false);
+  }
+
+  // Deals with issue #631, unstability of the Jool system's resonance.
+  private void Fix631() {
+    // Check whether this looks like stock.
+    if (FlightGlobals.Bodies.Count < 14) {
+      return;
+    }
+    Func<CelestialBody, double> mean_longitude =
+        (celestial) =>
+            (celestial.orbit.LAN + celestial.orbit.argumentOfPeriapsis) *
+                180 / Math.PI +
+            celestial.orbit.meanAnomalyAtEpoch;
+    CelestialBody jool = FlightGlobals.Bodies[8];
+    CelestialBody laythe = FlightGlobals.Bodies[9];
+    CelestialBody vall = FlightGlobals.Bodies[10];
+    CelestialBody bop = FlightGlobals.Bodies[11];
+    CelestialBody tylo = FlightGlobals.Bodies[12];
+    CelestialBody pol = FlightGlobals.Bodies[14];
+    bool is_stock = true;
+    is_stock &= jool.orbitingBodies.Count == 5;
+    is_stock &= jool.name   == "Jool";
+    is_stock &= laythe.name == "Laythe";
+    is_stock &= vall.name   == "Vall";
+    is_stock &= bop.name    == "Bop";
+    is_stock &= tylo.name   == "Tylo";
+    is_stock &= pol.name    == "Pol";
+    is_stock &= laythe.referenceBody == jool;
+    is_stock &= vall.referenceBody   == jool;
+    is_stock &= tylo.referenceBody   == jool;
+    is_stock &= (float)laythe.orbit.semiMajorAxis == 27184000f;
+    is_stock &= (float)vall.orbit.semiMajorAxis   == 43152000f;
+    is_stock &= (float)tylo.orbit.semiMajorAxis   == 68500000f;
+    is_stock &= (float)laythe.orbit.inclination == 0f;
+    is_stock &= (float)vall.orbit.inclination   == 0f;
+    is_stock &= (float)tylo.orbit.inclination   == 0.025f;
+    is_stock &= (float)mean_longitude(laythe) == 3.14f;
+    is_stock &= (float)mean_longitude(vall)   == 0.9f;
+    is_stock &= (float)mean_longitude(tylo)   == 3.14f;
+    if (!is_stock) {
+      return;
+    }
+    unmodified_orbits_[vall] = new Orbit(
+        vall.orbit.inclination,
+        vall.orbit.eccentricity,
+        laythe.orbit.semiMajorAxis * Math.Pow(2.472135954999579, 2.0 / 3.0),
```

Comment:
Are you sure you don't want to compute this number offline?  It could be sensitive to changes in Pow.